apps/web/vibes/soul/primitives/button/index.tsx

```diff
@@ -35,6 +35,10 @@ export interface ButtonProps extends ComponentPropsWithoutRef<'button'> {
  *   --button-ghost-text: hsl(var(--foreground));
  *   --button-ghost-border: transparent;
  *   --button-loader-icon: hsl(var(--foreground));
+ *   --button-danger-background: color-mix(in oklab, hsl(var(--error)), white 30%);
```

Comment:
Can you copy this css variables over to the `button.mdx` docs page as well under the `CSS Variables` section?

Reply:
@jamesqquick Done!